scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -146,7 +148,7 @@ workflow GvsCalculatePrecisionAndSensitivity {
         truth_vcf = truth_vcfs[i],
         truth_vcf_index = truth_vcf_indices[i],
         truth_bed = truth_beds[i],
-        interval_list = interval_list,
+        chromosomes = chromosomes,
```

Comment:
do we expect that we wont need to do an interval list in the future?

Reply:
it doesn't work with an interval list because `rtg vcfeval` expects a .bed file and falls over if you give it an interval list

Reply:
I dunno; in the past (wgs and exomes) we ran it based on whole chromosomes (first just 20, then everything but X and Y, and then everything).

Reply:
huh, I don't think I understand how an exome would look any good run over the whole genome, so I must be missing something. Were the high confidence truth intervals limited to just the exome? Otherwise everything outside of the exome would be a false negative and your recall would look very low.